### ISSUE TITLE
About randomly choosing images belong to 1000 categories

### ISSUE THREAD

**Issue by anonymous530** [dongyp13/Non-Targeted-Adversarial-Attacks#6]
Hello, thanks for your great work. I have met some question.
Does your image set just consist of 1000 images which belong to 1000 different categories? 
If so,  do I have tested the choosen images on different models?
Do you have some script to do so for convenient?
Thanks.

**Comment by dongyp13** [comment]
Hi,

We choose the images containing just one for each category from ImageNet validation set. They can also be correctly classified by the models. You can use the dataset from [https://github.com/tensorflow/cleverhans/tree/master/examples/nips17_adversarial_competition/dataset](https://github.com/tensorflow/cleverhans/tree/master/examples/nips17_adversarial_competition/dataset) for evaluation.

**Comment by anonymous530** [comment]
Thanks for your reply!
I have downloaded nips17_adversarial_competition/dataset before. But a few images in the dataset are empty. ("This photo is no longer available." ) Does it matter?

**Comment by dongyp13** [comment]
It's ok. The results may be different with those in the paper.

**Comment by anonymous530** [comment]
Ok. Thanks for your reply.

**Comment by hhhzzj** [comment]
Hi,
i have a question that i choose the images containing just one for each category from ImageNet validation set. They can also be correctly classified by ResNet-152. And i test the ASR, which is 99.6% for white box attack, and the ASR of black box attack to Inc-v3 net is 63.6%, which are higher than the paper's results. I can't understand the difference between my chosen dataset and yours.
Thank you so much!!!
